test/e2e/util.go

```diff
@@ -177,6 +177,10 @@ func Logf(format string, a ...interface{}) {
 	fmt.Fprintf(GinkgoWriter, "INFO: "+format+"\n", a...)
 }
 
+func Errorf(format string, a ...interface{}) {
+	fmt.Fprintf(GinkgoWriter, "EROR: "+format+"\n", a...)
```

Comment:
I only got four characters so that this lines up with INFO and STEP lines. I was thinking of ERRR?

Or maybe I should just not add this function?